### PR TITLE
grinnell.edu authentication added

### LINKED ISSUE #88: restrict access to @grinnell.edu emails

### PR DESCRIPTION
**Pull request by maripot**
Closes #88

### CODE REVIEW

**Comment by alexthemitchell** [comment]
Could you add a few tests?

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
   def set_current_user
     User.current_user = current_user
   end
-
+
```
Comment (alexthemitchell) [inline app/controllers/application_controller.rb]: Please remove this unnecessary whitespace.

**Comment by maripot** [comment]
devise runs the tests on its own - (class UserTest < ActiveSupport::TestCase) since we added :confirmable to devise User

**Comment by alexthemitchell** [comment]
Devise doesn't check that only @grinnell.edu emails are able to register; thats what should be tested.
